fs/copy_linux.go

```diff
@@ -27,7 +27,12 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-func copyFileInfo(fi os.FileInfo, name string) error {
```

Comment:
Why is it necessary to remove `os.FileInfo` from the interface and force an extra stat called? It is not an exported interface, adding the source string is fine if needed for Windows.

Reply:
Hi Derek,

I was on the fence about this and was hoping for feedback. If you are okay with adding the source string, I'll gladly add it.

Reply:
Done